packages/core/saved-objects/docs/openapi/bundled.yaml

```diff
@@ -62,6 +207,57 @@ paths:
         - url: https://localhost:5601
     servers:
       - url: https://localhost:5601
+  /api/encrypted_saved_objects/_rotate_key:
+    post:
+      summary: Rotate the encryption key for encrypted saved objects.
+      operationId: rotateEncryptionKey
+      description: |
+        Superuser role required. This functionality is in technical preview and may be changed or removed in a future release. Elastic will apply best effort to fix any issues, but features in technical preview are not subject to the support SLA of official GA features. NOTE: In most cases, total will be greater than successful even if failed is zero. The reason is that Kibana may not need or may not be able to rotate encryption keys for all encrypted saved objects. 
+      tags:
+        - saved objects
+      parameters:
+        - $ref: '#/components/parameters/kbn_xsrf'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                batch_size:
+                  description: Specifies a maximum number of saved objects that Kibana can process in a single batch. Bulk key rotation is an iterative process since Kibana may not be able to fetch and process all required saved objects in one go and splits processing into consequent batches. By default, the batch size is 10000, which is also a maximum allowed value.
+                  type: number
+                  default: 10000
+                type:
+                  description: Limits encryption key rotation only to the saved objects with the specified type. By default, Kibana tries to rotate the encryption key for all saved object types that may contain encrypted attributes.
+                  type: string
+            examples:
+              rotateEncryptionKeyRequest:
+                $ref: '#/components/examples/key_rotation_request'
```

Comment:
<img width="582" alt="Screenshot 2023-08-23 at 1 22 06 PM" src="https://github.com/elastic/kibana/assets/21210601/d4a896aa-dfc9-40ea-824b-78cc02b47bcc">

 This looks like an [example response](https://www.elastic.co/guide/en/kibana/current/saved-objects-api-rotate-encryption-key.html#saved-objects-api-rotate-encryption-key-example) , but it is listed under example request